modules/client-configuration/pages/clients-raspbian.adoc

```diff
@@ -0,0 +1,247 @@
+[[clients-raspbian]]
+= Registering {raspbian} Clients
+
+This section contains information about registering clients running {raspbian} operating systems.
+
+Bootstrapping can be used with {raspbian} clients for performing initial state runs, and for profile updates.
+
+ifeval::[{suma-content} == true]
+// Use list format: repository URLs / packages and metadata / version info
+[IMPORTANT]
+====
+* {raspbian} repository URLs are available from {scc}
+* Packages and metadata are provided by Raspbian OS, not by {suse}
+* For supported products, see the support table and the release notes
+====
+endif::[]
+
+== Prepare to Register
+
+Some preparation is required before you can register {raspbian} clients to the {productname} Server:
+
+* Ensure DNS is correctly configured and provides an entry for the client.
+  Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
+* The client must have the date and time synchronized with the {productname} Server before registration.
+
+
+
+== Add Software Channels
+
+Before you register {raspbian} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The architectures currently supported are: `arm64` and `armhf`.
+For full list of supported products and architectures, see xref:client-configuration:supported-features.adoc[].
+
+ifeval::[{suma-content} == true]
```

Comment:
Shouldn't we better comment this section for the moment, until it is actually supported by suma?